### ISSUE TITLE
Rename master branch from `main` to `master`

### ISSUE THREAD

**Issue by TimTinkers** [JossDuff/zero2prod#1]
Embrace the ways of yore and give thanks to the Old Gods of our forefathers, it's more based.

**Comment by JossDuff** [comment]
You have notifications on for whenever I create a new repo?  I'm blushing

**Comment by JossDuff** [comment]
Also, like my tweet it's a banger

**Comment by TimTinkers** [comment]
> You have notifications on for whenever I create a new repo? I'm blushing

No, I don't, I'm just also working late on a Friday night doing hyperfixated bullshit in Rust.
